shared/lib_pvshade.h

```diff
@@ -67,7 +67,7 @@ void diffuse_reduce(
 //	double phi0, // mask angle
 	double alb,
 	double nrows,
-
+	double skydiff0,
```

Comment:
Include detailed comment about what this is. See other comment about confusing variable name.